### PR TITLE
Update corp_locations.md

### PR DESCRIPTION
**Pull request by brianmadden**
Added some basic info to start the page.

### CODE REVIEW

File: docs/xmlapi/corporation/corp_locations.md [hunk 1]
```diff
@@ -1,4 +1,43 @@
...
+                <td>ids</td>
+                <td><strong>long</strong></td>
+                <td>ID of item to retrieve location information for</td>
+            </tr>
+		</tbody>
+	</table>
+
+* __Sample Output:__
+```
+<eveapi version="2">
```
Comment (JimiC) [inline docs/xmlapi/corporation/corp_locations.md]: Sample output should only contain the `result` section of the `xml`. The schema is explained in the intro page.

**Comment by deadlybulb** [comment]
LGTM, this looks like an easy merge

**Comment by Regner** [comment]
If you can go ahead and remove the parameters section we will get this pulled in. @CarbonAlabel removed the parameters that just had to do with authentication, since they are not endpoint specific, and added a section about authentication in #160.

**Comment by CarbonAlabel** [comment]
Cache timer and access mask are missing. Also, the layout would need some changes to be in line with other endpoint pages.

**Comment by Regner** [comment]
Thank you very much! :)

**Pull request merged**